src/main/java/org/zalando/nakadi/service/subscription/zk/CuratorZkSubscriptionClient.java

```diff
@@ -290,4 +291,23 @@ public void transfer(final String sessionId, final Collection<Partition.Partitio
             throw new NakadiRuntimeException(e);
         }
     }
+
+    @Override
+    public ZkSubscriptionNode getZkSubscriptionNode() {
```

Comment:
The name of the method is `get` but if I understood it correctly it updates the node. Maybe it should be renamed or operations should be separated between side effects and no side effects. Wdyt?